spec/bowling_score_spec.rb

```diff
@@ -0,0 +1,113 @@
+describe BowlingScore do
```

Comment:
All tests are clear and easy to read. Starting with the most simplest test helps understand what happens in the following tests.